### PR TITLE
MAUI-2436 Write the UG documentation for conditional styling feature

### PR DESCRIPTION
**Pull request by FarjanaParveen**
MAUI-2436 Write the UG documentation for conditional styling feature

### CODE REVIEW

File: MAUI/DataGrid/conditional-styling.md [hunk 1]
```diff
@@ -164,5 +164,218 @@ public class ColorConverter : IValueConverter
...
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional cell styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridcellstyle.png)
+
+## Style a cell based on MappingName and RowColumnIndex
+Styling can be applied to a particular cell based on RowIndex, ColumnIndex and MappingName property by writing the style for the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) TargetType.
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:202]: Styling can be applied to a particular cell based on RowIndex, ColumnIndex and MappingName property of the column by writing the style for the [DataGridCell](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCell.html) TargetType.

**Review by NeelakandanKannan — CHANGES_REQUESTED** [review]
Get back to me with the changes.

File: MAUI/DataGrid/conditional-styling.md (hunk 1 above)
Comment (FarjanaParveen) [reply MAUI/DataGrid/conditional-styling.md]: Hi @NeelakandanKannan , I have addressed your feedback

**Review by NeelakandanKannan — APPROVED** [review]
The changes are fine.

**Pull request merged**